test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/SignatureUtilityTests.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-#if NET46
```

Comment:
Are all the NET46 references removed from the code?

Reply:
In NuGet.Client, yes.  Some submodules use `NET46` (among others):

https://github.com/NuGet/FileSystem/blob/f1f3f0820a573b96b2faaf5b7e6be9a036e4c7aa/src/Microsoft.Extensions.FileSystemGlobbing/Util/StringComparisonHelper.cs#L22
https://github.com/aspnet/Extensions/blob/e6fac8061686c18531e2621ccef97dd5e0687a65/test/Microsoft.Extensions.Internal.Test/CopyOnWriteDictionaryHolderTest.cs#L7
https://github.com/aspnet/Extensions/blob/e6fac8061686c18531e2621ccef97dd5e0687a65/test/Microsoft.Extensions.Internal.Test/CopyOnWriteDictionaryHolderTest.cs#L16
https://github.com/aspnet/Extensions/blob/e6fac8061686c18531e2621ccef97dd5e0687a65/test/Microsoft.Extensions.Internal.Test/CopyOnWriteDictionaryTest.cs#L6
https://github.com/aspnet/Extensions/blob/e6fac8061686c18531e2621ccef97dd5e0687a65/test/Microsoft.Extensions.Internal.Test/CopyOnWriteDictionaryTest.cs#L15